### ISSUE TITLE
Generate contributors task is failing

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [typescript-eslint/typescript-eslint#4653]
## Overview

The most recent run, from this month: https://github.com/typescript-eslint/typescript-eslint/runs/5367805091?check_suite_focus=true

```
$ yarn ts-node --transpile-only ./tools/generate-contributors.ts && yarn all-contributors generate
$ /home/runner/work/typescript-eslint/typescript-eslint/node_modules/.bin/ts-node --transpile-only ./tools/generate-contributors.ts
$ /home/runner/work/typescript-eslint/typescript-eslint/node_modules/.bin/all-contributors generate
Cannot read property 'replace' of undefined
```

**Comment by JoshuaKGoldberg** [comment]
This doesn't happen locally for me. Unless someone has a hunch, investigation blocked on https://github.com/all-contributors/all-contributors/issues/588.

**Comment by armano2** [comment]
fixed by #5038
